Framework/Backend/log/InfoLoggerMessage.js

```diff
@@ -35,14 +38,13 @@ class InfoLoggerMessage {
 
   /**
    * Given a JSON object, parse through its values and return an InfoLoggerMessage with default values for missing ones
-   * @param {JSON} logJson - object with all or partial InfoLoggerMessage fields
+   * @param {object} logJson - object with all or partial InfoLoggerMessage fields
    * @returns {InfoLoggerMessage}
    */
   static fromJSON(logJson) {
     const log = new InfoLoggerMessage();
-    log._severity = logJson.severity && ['Info', 'Error', 'Fatal', 'Warning', 'Debug'].includes(logJson.severity) ?
-      logJson.severity : 'Info';
-    log._level = Number.isInteger(logJson?.level) ? logJson.level : 11;
+    log._severity = logJson.severity && LOG_SEVERITIES.includes(logJson.severity) ? logJson.severity : LogSeverity.Info;
+    log._level = Number.isInteger(logJson?.level) ? logJson.level : LogLevel.Developer;
```

Comment:
Should it not attempt to check if the provided value can be converted to a Number? Just in case a developer provides a string representation of a number?

Reply:
For me, in library we should expect users to provide proper type when it's properly typed (users means developpers using library, not end-users of course)